src/services/services.ts

```diff
@@ -3653,6 +3710,10 @@ namespace ts {
 
             let completionData = getCompletionData(fileName, position);
             if (!completionData) {
+                let entries = getJsDocCompletionEntries(fileName, position);

```

Comment:
how about adding a flag on the return of completionData, something like isJSDocTagName, then getcompletionEntry would fill in the tags if found. this way you can still return your symbols for non-jsdoc tag names, but still inside jsdoc